doc/aws/Setting_Up_IID_Attestation.md

```diff
@@ -0,0 +1,101 @@
+
+## Overview
+
+This walkthrough will guide you through steps to install and configure SPIRE Server and SPIRE Agent for AWS IID based attestation.
+Follow instructions [ here ](https://github.com/spiffe/spiffe-example/blob/master/ec2/README.md) for a basic demo VPC and EC2 setup.
+
+### Installing SPIRE Server/Agent:
+
+Both SPIRE Server and SPIRE Agent binaries are bundled in the same distribution and can be found [ here ](https://github.com/spiffe/spire/releases/latest).
+Download and extract the SPIRE distribution into ~/opt directory:
+
+        mkdir ~/opt
+        wget https://github.com/spiffe/spire/releases/download/<release_version>/spire-<release_version>-linux-x86_64-glibc.tar.gz
+        tar -C ~/opt -xzf spire-<release_version>-linux-x86_64-glibc.tar.gz
+
+### Installing aws IID attestor plugin on SPIRE server:
```

Comment:
This is really "building" as well as installing. Suggest breaking it our accordingly.